### ISSUE TITLE
uninitialized constant Socket::MSG_NOSIGNAL

### ISSUE THREAD

**Issue by Esity** [swisscom/ruby-netsnmp#60]
Looks like the newer versions of netsnmp are complaining about `Socket::MSG_NOSIGNAL` 

By downgrading to 0.4.0, this error goes away. Confirmed this on both ruby 2.7.3 and 3.0.1

```ruby
manager.get(oid: "sysName.0")


# uninitialized constant Socket::MSG_NOSIGNAL
# Did you mean?  Socket::MSG_HOLD
```

**Comment by HoneyryderChuck** [comment]
Hi @Esity ,

Thx for the bug report.


I'm assuming you're not working with Linux. Can you give more information about your system? I'm mostly interested in the following (output is from my system, you can reply with yours):

```ruby
# in irb
require "socket"

Socket::constants.select{|c|c.to_s.start_with?("MSG_")}
#=> [:MSG_DONTWAIT, :MSG_OOB, :MSG_PEEK, :MSG_DONTROUTE, :MSG_EOR, :MSG_TRUNC, :MSG_CTRUNC, :MSG_WAITALL, :MSG_EOF, :MSG_FLUSH, :MSG_HOLD, :MSG_SEND, :MSG_HAVEMORE, :MSG_RCVMORE]

RUBY_PLATFORM
#=> "x86_64-darwin20"
```

**Comment by HoneyryderChuck** [comment]
Fixed with  5c1a797bff67973491025e59ce7a5eb314e52e69 . Just released 0.6.1 .

**Comment by Esity** [comment]
Amazing. Thanks for the fast turnaround. Will test this tonight or tomorrow.
